src/mongroupdef.cpp

```diff
@@ -24,6 +24,11 @@ MonsterGroupResult MonsterGroupManager::GetResultFromGroup(
 {
     int spawn_chance = rng(1, 1000);
     MonsterGroup group = GetMonsterGroup( group_name );
+    int replace_time = DAYS(group.monster_group_time * ACTIVE_WORLD_OPTIONS["MONSTER_GROUP_DIFFICULTY"]) * (calendar::turn.season_length() / 14);
```

Comment:
Applying it to existing monsters is the way it should work eventually, this
approach is ok for now, but will probably be dismantled when that version
is implemented.